Must import `wkt` crate if any message is present

We need to import the `wkt` crate to define the `wkt::message::Message` trait. Otherwise we get errors:

```
error[E0433]: failed to resolve: use of undeclared crate or module `wkt`
  --> src/generated/cloud/language/v2/src/model.rs:85:6
   |
85 | impl wkt::message::Message for Document {
   |      ^^^ use of undeclared crate or module `wkt`
```


Note that one of the golden files already has this problem. It uses the crate here:

https://github.com/googleapis/google-cloud-rust/blob/25149ad9f5cfdef32822a29710a519a49e37807b/generator/testdata/rust/protobuf/golden/type/src/model.rs#L117

But it is missing here:


https://github.com/googleapis/google-cloud-rust/blob/25149ad9f5cfdef32822a29710a519a49e37807b/generator/testdata/rust/protobuf/golden/type/src/model.rs#L20-L21

--- a/.sidekick.toml
+++ b/.sidekick.toml
@@ -41,11 +41,13 @@ version = "0.1.0"
 'package:gax'         = 'used-if=services,package=gcp-sdk-gax,path=src/gax,feature=unstable-sdk-client,version=0.1'
 # Only used if LROs are present
 'package:lro' = 'used-if=lro,package=gcp-sdk-lro,path=src/lro,version=0.0'
+# I (coryan@) got lazy, it is tedious to auto-detect if this is used in `sidekick`.
+# OTOH, the only case where this is not used is a crate without any messages, i.e., just enums.
+'package:wkt' = 'force-used=true,package=gcp-sdk-wkt,path=src/wkt,source=google.protobuf,version=0.1'
 # These are crates in `google-cloud-rust`. If not used, `sidekick` prunes them
 # from the list of depedencies.
 'package:api'         = 'package=gcp-sdk-api,source=google.api,path=src/generated/api,version=0.1'
 'package:gtype'       = 'package=gcp-sdk-type,source=google.type,path=src/generated/type,version=0.1'
-'package:wkt'         = 'package=gcp-sdk-wkt,path=src/wkt,source=google.protobuf,version=0.1'
 'package:iam_v1'      = 'package=gcp-sdk-iam-v1,source=google.iam.v1,path=src/generated/iam/v1,version=0.1'
 'package:location'    = 'package=gcp-sdk-location,source=google.cloud.location,path=src/generated/cloud/location,version=0.1'
 'package:longrunning' = 'package=gcp-sdk-longrunning,source=google.longrunning,path=src/generated/longrunning,version=0.1'

